.semaphore/semaphore.yml

```diff
@@ -0,0 +1,20 @@
+version: v1.0
+name: calico docs CI
+agent:
+  machine:
+    type: e1-standard-2
+    os_image: ubuntu1804
+
+execution_time_limit:
+   hours: 1
+
+blocks:
+  - name: validation
+    dependencies: []
+    task:
+      jobs:
+        - name: CI
+          commands:
+            - checkout
+            - make dev-manifests
```

Comment:
@lwr20 I suggest removing the dev-manifests target for now so we can get this merged. That target would be great to include, but I think is not MVP for CI on this repo.